python/ray/data/read_api.py

```diff
@@ -105,6 +105,8 @@ def from_items(
     items: List[Any],
     *,
     parallelism: int = -1,
+    concurrency: Optional[int] = None,
```

Comment:
`from_items` doesn't launch tasks, so `concurrency` wouldn't do anything. Should we remove it?

Reply:
ah good catch, let me update.

Reply:
other "from_xxx" APIs don't need this either.

Reply:
yes, agree. Will remove for all `from` APIs. Missed that.